apps/admin/backend/schema.sql

```diff
@@ -72,7 +74,8 @@ create table ballot_images (
   image blob not null,
   layout text not null,
   foreign key (cvr_id) references cvrs(id)
-    on delete cascade
+    on delete cascade,
+  unique(cvr_id, is_back)
```

Comment:
Enums might be tricky in SQLite but curious whether you considered something like `front_or_back` (where the possible values are `"front"` and `"back"`) instead of a boolean `is_back`.

Just thinking about cases where we might have to check that a sheet is a front sheet with a slightly confusing `!is_back` condition.

Regardless, I'm okay with `is_back`!

Reply:
I did think about it and I had a reason but now I can't remember it... Will circle back and change this if I don't think of it.